src/ImageMap/TecanLayout.tsx

```diff
@@ -0,0 +1,199 @@
+  key: number;
+  content?: React.ReactElement | null;
+  position: React.SVGProps<SVGRectElement>;
+};
+
+const TOP_LABWARES_Y_POSITION = 115;
+const MIDDEL_LABWARES_Y_POSITION = 170;
+const LOWER_LABWARES_Y_POSITION = 225;
+
+const LEFT_LABWARES_X_POSITION = 233;
+const CENTER_LABWARES_X_POSITION = 317;
+const RIGHT_LABWARES_X_POSITION = 399;
+
+const MM_LABWARE_POSITION: React.SVGProps<SVGRectElement> = {
+  width: 35,
+  height: 190,
+  x: 192,
+  y: 98,
+};
+
+const PLATE_SIZE_LANDSCAPE_MODE: React.SVGProps<SVGRectElement> = {
+  width: 75,
+  height: 50,
+};
+
+const PLATE_SIZE_PORTRAIT_MODE = {
+  width: 50,
+  height: 75,
+};
+
+const A_PLATE_LABWARE_POSITION: React.SVGProps<SVGRectElement> = {
+  ...PLATE_SIZE_LANDSCAPE_MODE,
+  x: LEFT_LABWARES_X_POSITION,
+  y: TOP_LABWARES_Y_POSITION,
+};
+const B_PLATE_LABWARE_POSITION: React.SVGProps<SVGRectElement> = {
+  ...PLATE_SIZE_LANDSCAPE_MODE,
+  x: LEFT_LABWARES_X_POSITION,
+  y: MIDDEL_LABWARES_Y_POSITION,
+};
+const NEMO_WATER_LABWARE_POSITION: React.SVGProps<SVGRectElement> = {
+  ...PLATE_SIZE_LANDSCAPE_MODE,
+  x: LEFT_LABWARES_X_POSITION,
+  y: LOWER_LABWARES_Y_POSITION,
+};
+const NEMO_DILUTION_LABWARE_POSITION: React.SVGProps<SVGRectElement> = {
+  ...PLATE_SIZE_LANDSCAPE_MODE,
+  x: CENTER_LABWARES_X_POSITION,
+  y: TOP_LABWARES_Y_POSITION,
+};
+const NEMO_DEST_PCR_2_LABWARE_POSITION: React.SVGProps<SVGRectElement> = {
+  ...PLATE_SIZE_LANDSCAPE_MODE,
+  x: CENTER_LABWARES_X_POSITION,
+  y: MIDDEL_LABWARES_Y_POSITION,
+};
+const NEMO_DEST_TAQ_MAN_LABWARE_POSITION: React.SVGProps<SVGRectElement> = {
+  ...PLATE_SIZE_LANDSCAPE_MODE,
+  x: CENTER_LABWARES_X_POSITION,
+  y: LOWER_LABWARES_Y_POSITION,
+};
+
+const DEST_PCR_LABWARE_POSITION: React.SVGProps<SVGRectElement> = {
+  ...PLATE_SIZE_LANDSCAPE_MODE,
+  x: RIGHT_LABWARES_X_POSITION,
+  y: TOP_LABWARES_Y_POSITION,
+};
+const DEST_LC_LABWARE_POSITION: React.SVGProps<SVGRectElement> = {
+  ...PLATE_SIZE_LANDSCAPE_MODE,
+  x: RIGHT_LABWARES_X_POSITION,
+  y: MIDDEL_LABWARES_Y_POSITION,
+};
+const FLUID_XLABWARE_POSITION: React.SVGProps<SVGRectElement> = {
+  ...PLATE_SIZE_LANDSCAPE_MODE,
+  x: RIGHT_LABWARES_X_POSITION,
+  y: LOWER_LABWARES_Y_POSITION,
+};
+
+const DEST_PCR_1_LABWARE_POSITION: React.SVGProps<SVGRectElement> = {
+  ...PLATE_SIZE_PORTRAIT_MODE,
+  x: 492,
+  y: 118,
+};
+const DEST_PCR_2_LABWARE_POSITION: React.SVGProps<SVGRectElement> = {
+  ...PLATE_SIZE_PORTRAIT_MODE,
+  x: 492,
+  y: 192,
+};
+
+export function TecanLayout(props: {
+  src: string;
+  labwares: {
+    mmPlate?: Labware;
+    destPcr?: Labware;
+    aPlate?: Labware;
+    bPlate?: Labware;
+    nemoWater?: Labware;
+    nemoDilution?: Labware;
+    nemoDestPcr2?: Labware;
+    nemoDestTaqMan?: Labware;
+    destLc?: Labware;
+    fluidX?: Labware;
+    destPcr1?: Labware;
+    destPcr2?: Labware;
+  };
+}) {
+  const labwaresWithPosition: Array<LabwareWithPositon> = [
+    {
+      key: 0,
+      content: props.labwares.mmPlate?.content ?? null,
```

Comment:
```suggestion
      content: props.labwares.mmPlate?.content,
```

Both `null` and `undefined` should be fine